### PR TITLE
fail tolerant bbox projection. might be empty...

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[fail tolerant bbox projection.](https://trello.com/c/MbEYMcHK/113-fail-tolerant-bbox-projection)

File: datatypes/src/operations/reproject.rs [hunk 1]
```diff
@@ -208,16 +208,20 @@ where
...
             .chain(lower_line)
             .chain(left_line)
             .collect();
 
-        // TODO: this should use the fail_tolarant reprojection and fail only if there are not enough coordinates after reprojection. This could also require us to use a grid instead of bounds. The good news is that the Grid can already provide this.
-        MultiPoint::new_unchecked(cs)
-            .reproject(projector)
-            .map(|mp| mp.spatial_bounds())
+        let proj_outline_coordinates: Vec<Coordinate2D> =
+            project_coordinates_fail_tolarant(&outline_coordinates, projector)
```
Comment (michaelmattig) [inline datatypes/src/operations/reproject.rs:218]: typo in function name: tolarant -> tolerant. Even better name would be fault_tolerant

**Comment by michaelmattig** [comment]
grafik
WGS84->Webmercator works but if I add NDVI3857 data set to a WGS84 map the top ans bottom of the map still has error tiles that say

> Operator: DataTypeError: ProjInternal error: Conversion("tolerance condition error")

**Review by michaelmattig — CHANGES_REQUESTED** [review]
cf my comments

**Comment by michaelmattig** [comment]
noch eine Sache: bei Vector Data, konkret Natural Earth 10m Ports 3857 gibts auch nen Fehler wenn man in der Karte zu weit rauszoomt

> Operator: DataTypeError: ProjInternal error: Conversion("tolerance condition error")

**Pull request merged**